src/_layouts/page.html

```diff
@@ -90,41 +90,29 @@
     <!-- End Google Tag Manager (noscript) -->
 
     <header>
-      <nav class="dark-bg">
-        <input type="checkbox" />
-        <label>&equiv;</label>
-        <a class="home-button" href="/" aria-label="BeakBox Home page">
-          <img class="clickable" alt="Logo" src="/imgs/title.svg" />
+      <nav class="nav">
```

Comment:
This class name is redundant to the element. If using the element by itself is too vague, `header > nav` may work instead or a clearer class name.